config/test_config.toml

```diff
@@ -1,6 +1,7 @@
 SQLALCHEMY_DATABASE_URI = "postgresql://geonatadmin:geonatpasswd@127.0.0.1:5432/geonature2db"
 URL_APPLICATION = 'http://127.0.0.1:4200'
 API_ENDPOINT = 'http://127.0.0.1:8000'
-API_TAXHUB = 'http://127.0.0.1:5000/api'
+API_TAXHUB = 'https://demo.geonature.fr/taxhub/api'
```

Comment:
Il ne faut pas tester sur la demo.
Cela ne marchait pas en local ?

Reply:
Non ça ne fonctionnait pas, j'avais l'impression que l'instance de Taxhub qui était installée ne fonctionnait pas... 
Et je trouve ça dommage de devoir installer Taxhub. 
A voir si avec la PR de dockerisation de Taxhub, on ne pourrait pas utiliser l'image docker pour la monter dans un conteneur dans le GitHub action. Mais ça demanderait plus de boulot...

Reply:
L'API de Taxhub est déjà installée avec GeoNature. 
Il y a même un test pour voir si elle est UP : https://github.com/PnX-SI/GeoNature/blob/Adrien-Pajot-from-dev-to-cypress-test/.github/workflows/test.yml#L244
si ce test passe il n'y a pas de raison que l'API ne réponde pas ..

Reply:
C'était pour les tests frontend que vous aviez dû faire ça ?

Reply:
Ah oui effectivement, désolé, je n'avais pas vu que Taxhub était dans les dependencies...

En regardant le dernier run, ce test n'a pas l'air de passer mais aucun code d'erreur n'a l'air d'être renvoyé : 
```
Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.9.12/x64/lib/python3.9/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/opt/hostedtoolcache/Python/3.9.12/x64/lib/python3.9/site-packages/flask/cli.py", line 184, in find_app_by_string
    app = call_factory(script_info, attr, args)
  File "/opt/hostedtoolcache/Python/3.9.12/x64/lib/python3.9/site-packages/flask/cli.py", line 119, in call_factory
    return app_factory()
  File "/home/runner/work/GeoNature/GeoNature/backend/dependencies/TaxHub/apptax/app.py", line 78, in create_app
    th_app = Application.query.filter_by(code_application='TH').one()
  File "/opt/hostedtoolcache/Python/3.9.12/x64/lib/python3.9/site-packages/sqlalchemy/orm/query.py", line 3500, in one
    raise orm_exc.NoResultFound("No row was found for one()")
sqlalchemy.orm.exc.NoResultFound: No row was found for one()
```

Reply:
Oui c'est pour les tests frontend

Reply:
Effectivement, TaxHub n'est pas renseigné dans TModules. 
Quoi qu'il en soit il faut qu'il tourne en local .. je regarde

Reply:
Il fallait juste monter la branche alembic "taxhub" et ça roule !
https://github.com/PnX-SI/GeoNature/commit/6c936b09304fb802c1b08a9270b4409f02c530f8

Reply:
Ahh ok ! Du coup tu fais la modif direct ? Ou je le fais ?

Reply:
Je l'ai fait sur ma branche de modif. Je mergerai directement celle là